third_party/amd/lib/TritonAMDGPUTransforms/RefineOps.cpp

```diff
@@ -219,80 +218,200 @@ struct DotOpMFMAConverter {
 
     const auto numRepM = repA[1];
     const auto numRepN = repB[2];
+    const auto numRepK = repA[2];
     const auto numRepB = repA[0];
-
-    LDBG("numRepM: " << numRepM << "; numRepN: " << numRepN
-                     << "; numRepB: " << numRepB);
-
-    constexpr int M = 0;
-    constexpr int N = 1;
-
-    SmallVector<int64_t> refinedShapeA = {shapeA[M] / numRepM, shapeA[N]};
-    SmallVector<int64_t> refinedShapeB = {shapeB[M], shapeB[N] / numRepN};
-    SmallVector<int64_t> refinedShapeCD = {shapeC[M] / numRepM,
-                                           shapeC[N] / numRepN};
-
-    SmallVector<int64_t> elementsPerSlice = {refinedShapeCD[0],
-                                             refinedShapeCD[1]};
-
-    auto refinedTensorTypeA =
-        RankedTensorType::get(refinedShapeA, elemTyA, encodeA);
-    auto refinedTensorTypeB =
-        RankedTensorType::get(refinedShapeB, elemTyB, encodeB);
-    auto refinedTensorTypeC =
-        RankedTensorType::get(refinedShapeCD, elemTyC, encodeC);
-    auto refinedTensorTypeD =
-        RankedTensorType::get(refinedShapeCD, elemTyD, encodeD);
-
-    constexpr bool mutableMemory = true;
-    auto sharedMemorySpace = triton::gpu::SharedMemorySpaceAttr::get(ctx);
-
+    SmallVector<int64_t> numRepShape = {numRepM, numRepN, numRepK};
+    DBGS() << "Dot-Tiling\n";
+    DBGS() << "totalReps: " << numRepShape[0] << "x" << numRepShape[1] << "x"
```

Comment:
Must be

```cpp
LDBG("Dot-Tiling");
LDBG("totalReps: " << numRepShape[0] << "x" << numRepShape[1] << "x");
```

There are few place that need to be addressed in the same way

Reply:
done